core/keys/symbols.py

```diff
@@ -0,0 +1,122 @@
+# define the spoken forms for symbols in command and dictation mode
+symbols_command_and_dictation = {}
+
+# for dragon, we add a couple of mappings that don't work for conformer
+# i.e. dragon supports some actual symbols as the spoken form
+dragon_symbols_command_and_dictation = {
+    "`": "`",
+    ",": ",",
+}
+
+# define the spoken forms for symbols that are intended for command mode only
+symbols_command = {}
+
+# define spoken form for symbols for use in create_spoken_forms.py functionality
+# we define a handful of symbol only. at present, this is restricted to one entry per symbol.
+symbols_for_create_spoken_forms = {
+    # for application names like "Movies & TV"
+    "and": "&",
+    # for emails
+    "at": "@",
+    "dot": ".",
+    # for application names like "notepad++"
+    "plus": "+",
+}
+
+
+class Symbol:
+    character: str
+    command_forms: list[str] = None
+    command_and_dictation_forms: list[str] = None
+
+    def __init__(
+        self, character: str, command_and_dictation_forms=None, command_forms=None
+    ):
+        self.character = character
+
+        if command_and_dictation_forms:
+            self.command_and_dictation_forms = (
+                [command_and_dictation_forms]
+                if isinstance(command_and_dictation_forms, str)
+                else command_and_dictation_forms
+            )
+
+        if command_forms:
+            self.command_forms = (
+                [command_forms] if isinstance(command_forms, str) else command_forms
+            )
+
+
+currency_symbols = [
+    Symbol("$", ["dollar sign"], ["dollar"]),
+    Symbol("£", ["pound sign"], ["pound"]),
+]
+
+symbols = [
```

Comment:
We're not sure that the data class where we combined punctuation and symbols is easy to read. Also we really need to read through this to make sure we don't have any regression from the current setup.

Reply:
We agreed in the subsequent session that this is probably the least-worst option.
https://github.com/talonhub/community/pull/1763#discussion_r1986123977